crates/spirv-builder/README.md

```diff
@@ -0,0 +1,47 @@
+<!-- inline html -->
+<!-- markdownlint-disable-file MD033 -->
+# `spirv-builder`
+
+![Rust version](https://img.shields.io/badge/rust-1.66.0_nightly--2022--10--29-purple.svg)
```

Comment:
I think the 1.66.0 is an accident? Or at least it might be misleading to ppl who expect a stable version.